content/guides/typescript.md

```diff
@@ -67,28 +83,25 @@ module.exports = {
  },
  output: {
    filename: 'bundle.js',
-   path: __dirname
+   path: path.resolve(__dirname, 'dist');
  }
 };
 ```
 
-This will direct webpack to _enter_ through `./index.ts`, _load_ all `.ts` and `.tsx` files through the `ts-loader`, and _output_ a `bundle.js` file in our current directory.
-
+This will direct webpackgit sdt to _enter_ through `./index.ts`, _load_ all `.ts` and `.tsx` files through the `ts-loader`, and _output_ a `bundle.js` file in our current directory.
 
-## Loaders
 
-The following loaders for TypeScript:
+## Loader
 
-- [`awesome-typescript-loader`](https://github.com/s-panferov/awesome-typescript-loader)
-- [`ts-loader`](https://github.com/TypeStrong/ts-loader)
+[`ts-loader`](https://github.com/TypeStrong/ts-loader)
 
-Awesome TypeScript Loader has created a [wonderful explanation](https://github.com/s-panferov/awesome-typescript-loader#differences-between-ts-loader) of the difference between `awesome-typescript-loader` and `ts-loader`.
-
-We chose to use `ts-loader` in this guide as it makes enabling additional webpack features, such as importing other web assets, a bit easier.
+We will use `ts-loader` in this guide as it makes enabling additional webpack features, such as importing other web assets, a bit easier.
 
 
 ## Source Maps
 
+To learn more about Source Maps, see the [development guide](/guides/development.md).
+
 To enable source maps, we must configure TypeScript to output inline source maps to our compiled JavaScript files. The following line must be added to our `tsconfig.json`:
 
 ``` json
```

Comment:
To keep it consistent, please make this a diff of your entire file. It's easier to read for beginners.